telegraf-1.26.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: telegraf-1.26
+  version: 1.26.3
+  epoch: 1
```

Comment:
Does the epoch needs to be set to `0` as this is a considered as a new package

Reply:
I think you are right as this is effectively a new package, but I used the original epoch out of an abundance of caution.  Changed to zero now.